README.md

```diff
@@ -1,87 +1,20 @@
-<p align="center"><img src="https://cloud.githubusercontent.com/assets/3129129/22811426/bb69dc06-ef0c-11e6-8092-a0bea9060b35.png"/></p>
+# cra-template-nimble
```

Comment:
Should we add the [license and about section](https://github.com/nimblehq/rails-templates#license) in this PR or create another task to modify Readme?

Reply:
@carryall there's a topmost section before [Get started](https://github.com/nimblehq/rails-templates#get-started) with Nimble logo and project Name. We can add it too.

Reply:
✅  added in [fa2a7a0](https://github.com/nimblehq/react-templates/pull/12/commits/fa2a7a0ebd6fad3dac3e3dbd79bd67e4b9ff55b1)